packages/core/addon/models/delivery-rule.js

```diff
@@ -37,6 +37,7 @@ export default DS.Model.extend(Validations, {
   format: DS.attr({ defaultValue: () => {} }),
   recipients: DS.attr({ defaultValue: () => [] }),
   version: DS.attr('number', { defaultValue: '1' }),
+  notifyWhenEmpty: DS.attr('number', { defaultValue: '0' }),
```

Comment:
can we use `boolean` instead of `number`?

Reply:
Actually this needs to be in scheduling rules, not it's own property.

Reply:
Can you explaining a little more on why we need to put this under schedulingRules instead of at the same level of format, frequency, and recipients?